vampyr/.pytest_cache/v/cache/nodeids

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
You should remove the `.pytest_cache` folder that was committed by accident. In the `.gitignore` of the root folder ass `.pytest_cache` so the folder is simply ignored in the future.